File: resourcespoet/src/test/java/com/commit451/resourcespoet/IntegerArrayTest.java
package com.commit451.resourcespoet;

import org.junit.Test;

import java.util.ArrayList;

/**
 * Tests the resources creation
 */
public class IntegerArrayTest {

    @Test
    public void integerArrayTest() throws Exception {
        ArrayList<Integer> numbers = new ArrayList<>();
        numbers.add(1);
        numbers.add(2);

        ResourcesPoet poet = ResourcesPoet.create()
                .addIntegerArray("numbers", numbers);

        TestUtil.assertEquals("integer_array.xml", poet);
    }
}

File: src/test/kotlin/com/commit451/resourcespoet/IntegerArrayTest.kt
package com.commit451.resourcespoet

import org.junit.Test

/**
 * Tests the resources creation
 */
class IntegerArrayTest {

    @Test
    fun integerArrayTest() {
        val numbers = ArrayList<Int>()
        numbers.add(1)
        numbers.add(2)

        val poet = ResourcesPoet.create()
            .addIntegerArray("numbers", numbers)

        TestUtil.assertEquals("integer_array.xml", poet)
    }
}
